.github/workflows/pull_request.yml

```diff
@@ -0,0 +1,33 @@
+name: pull-request
+
+on:
+  pull_request:
+    types: [opened, reopened, synchronize]
+
+concurrency: 
+  group: ${{ github.head_ref }}
+  cancel-in-progress: true
+
+jobs:
+  ci:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+
+    - name: Install flutter
+      uses: subosito/flutter-action@v2
+      with:
+        channel: stable
+
+    - name: Get dependencies
+      run: flutter pub get
+
+    - name: Run formatter
+      run: flutter format --set-exit-if-changed .
```

Comment:
this doesn't actually format, it just checks if everything is properly formatted.